Makefile

```diff
@@ -1,4 +1,5 @@
-JAEGER_IMPORT_PATH=github.com/jaegertracing/jaeger
+SHELL := /bin/bash
```

Comment:
is this needed?

Reply:
Yes, because regex matches via the `=~` operator requires `[[ ... ]]`, which is only supported in `bash`, and not `sh`. Without it, the following error will result: `/bin/sh: 4: [[: not found`.